src/interdiff.c

```diff
@@ -58,6 +59,27 @@
 #define PATCH "patch"
 #endif
 
+#define COLOR_RESET     "\033[0m"
+
+/* Line type for coloring */
+enum line_type {
+	LINE_FILE = 0,
+	LINE_HEADER,
+	LINE_HUNK,
+	LINE_ADDED,
+	LINE_REMOVED,
```

Comment:
Add an entry at the end named `LINE_MAX` (without a trailing comma).

Reply:
I named it LINE_TYPE_MAX.